### ISSUE TITLE
the post &put request does not work

### ISSUE THREAD

**Issue by cnxuyc** [senecajs/seneca-web-adapter-koa2#15]
my route map
```
module.exports = [
    {
        prefix: 'v1/user',
        pin: 'area:user,action:*',
        map: {
            create: {
                POST: true
            },
            list: {
                GET: true,
                POST: true
            },
        }
    }
];

```
when i modify the code like this ,it is work ;please check it

```
        if (['POST', 'PUT'].indexOf(ctx.req.method) > -1) {
          body = await Parse(ctx)
        }
```
to
```
if (['POST', 'PUT'].indexOf(ctx.request.method) > -1) {
          body = Object.assign({}, ctx.request.body)
        }
```

**Comment by tswaters** [comment]
Hi, sorry about the wait on this.

There is an option you can pass to `seneca-web` options called `parseBody` that will tell the adapter to not attempt to parse the body of the request.

For details see the following section of the seneca-web readme:
https://github.com/senecajs/seneca-web/blob/ac286c7/README.md#body-parser

That's all fine and good, but this adapter didn't actually support the option properly.... I'm fixing that now, should be available in the next patch version, v1.0.6 once I finish.

**Comment by tswaters** [comment]
Apologies for the wait.... here's an update:

This change turned out to be a bit loaded....  not because of the change, but upstream koa modules have started using destructuring, and our CI is setup to run against node4 which doesn't support the syntax.

After thinking about it, I think the best way forward is to rip out all the babel stuff, update CI to run against node8 and publish a major version update of the module that uses native async/await.

The real question is if I patch 1.x to include this (I probably should) and do a major version afterwards.  I really just need time to do this.... it is on my radar.

**Comment by tswaters** [comment]
Sorry about the wait on this one - fixed in v1.1.0
